README.md

```diff
@@ -1,15 +1,93 @@
 # Temporal Ruby SDK
 
-![CI status](https://github.com/temporalio/sdk-ruby/actions/workflows/ci.yml/badge.svg?branch=main)
+[![CI status](https://github.com/temporalio/sdk-ruby/actions/workflows/ci.yml/badge.svg?branch=main)](https://github.com/temporalio/sdk-ruby/actions/workflows/ci.yml?query=branch%3Amain)
+[![Build](https://github.com/temporalio/sdk-ruby/actions/workflows/build-native-ext.yml/badge.svg)](https://github.com/temporalio/sdk-ruby/actions/workflows/build-native-ext.yml?query=branch%3Amain+event%3Arelease)
+[![Gem Version](https://badge.fury.io/rb/temporalio.svg)](https://rubygems.org/gems/temporalio)
 
-Temporal is a distributed, scalable, durable, and highly available orchestration engine used to
-execute asynchronous long-running business logic in a scalable and resilient way.
+[Temporal](https://temporal.io/) is a distributed, scalable, durable, and highly available
+orchestration engine used to execute asynchronous long-running business logic in a scalable and
+resilient way.
 
 "Temporal Ruby SDK" is the framework for authoring workflows and activities using the Ruby
 programming language.
 
 ⚠️ UNDER DEVELOPMENT
 
+The Ruby SDK is under development. There are no compatibility guarantees at this time.
+
+At this point the SDK only supports the **Temporal Client** capabilities:
+
+- Starting a workflow (defined in any other SDK)
+- Interacting with a workflow (cancelling, querying, signalling, etc)
+- Interceptor and Data Conversion support
+- gRPC access to Temporal Server
+
+
+## Quick Start
+
+### Installation
+
+Add the [`temporalio` gem](https://rubygems.org/gems/temporalio) to your Gemfile:
+
+```ruby
+gem 'temporalio'
+```
+
+The SDK is now ready for use. To build from source, see [Dev Setup](#dev-setup).
+
+**NOTE: This README is for the current branch and not necessarily what's released on RubyGems.**
+
+
+## Usage
+
+### Client
+
+A client can be created and used to start a workflow like so:
+
+```ruby
+# Establish a gRPC connection to the server
+connection = Temporal::Connection.new('http://localhost:7233')
```

Comment:
nit: please make it `localhost:7233`, without the http prefix. That's the format we use pretty much everywhere in doc and other SDKs.

Reply:
:+1: We need to avoid thinking that it's http (even though that's what core accepts)

Reply:
Good point 👍 Added this to the other PR (as it's more relevant) — https://github.com/temporalio/sdk-ruby/pull/69/commits/9c2dca303118f0e136adde35a617b781553bdb4b